core/src/main/java/com/netease/arctic/table/BasicKeyedTable.java

```diff
@@ -18,6 +18,7 @@
 
 package com.netease.arctic.table;
 
+import com.google.common.collect.Maps;
```

Comment:
Nit: Use the `org.apache.iceberg.relocated.com.google.xxx`.

Reply:
fixed